### DISCUSSION TITLE
Accessibility

### DISCUSSION THREAD

**Discussion by kieranlittle667** [ciderapp/Cider#993] [Added Features]
Hi,
Love the app. However, it is not fully accessible with a screenreader for blind / visually impaired users. There are many unlabeled buttons and unlabeled elements and I found navigating the app with a keyboard to be quite cumbersome.
Could this be looked into to make the app usable?
Thanks.
Thanks.

**Comment by Amaru8** [comment]
We are glad you like Cider! Tracking this issue in #876.

**Reply by kieranlittle667** [reply]
Thanks for your reply, I did scan through the issues but didn't see this one.

**Comment by Amaru8** [comment]
Added in #1016, locking discussion.
